src/wcag-link-data.ts

```diff
@@ -0,0 +1,104 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { DictionaryStringTo } from './dictionary-types';
+
+export interface WCAGLinkData {
+    text: string;
+    url?: string;
+    title?: string;
+}
+
+function standardWcag21Link(
+    urlFormattedTitle: string,
+    sectionNumber: string,
+    title: string,
+): WCAGLinkData {
+    return {
+        text: `WCAG ${sectionNumber}`,
+        title: title,
+        url: `https://www.w3.org/WAI/WCAG21/Understanding/${urlFormattedTitle}`,
+    };
+}
+
+// prettier-ignore
+export const axeTagsToWcagLinkData: DictionaryStringTo<WCAGLinkData> = {
+    wcag111: standardWcag21Link("non-text-content", '1.1.1', 'Non-text Content'),
```

Comment:
Is it intentional to have double quotes for `urlFormattedTitle` but single quotes for the other strings?

Reply:
No, fixed